src/shared/utils.tsx

```diff
@@ -19,11 +19,17 @@ export const formatTime = (oldDay: string): string => {
 export const modifyDatetime = (oldDay: number): string => {
 
   const inputDateTime: DateTime = DateTime.fromMillis(oldDay);
-  const diffInMilli: number = inputDateTime.diffNow().valueOf();
-  const dayInMilli: number = Duration.fromObject({ days: 1 }).valueOf();
+  
+  // const diffInMilli: number = inputDateTime.diffNow().valueOf();
+  // const dayInMilli: number = Duration.fromObject({ days: 1 }).valueOf();
+  // return diffInMilli >= dayInMilli ? (inputDateTime.toRelativeCalendar() || "").split(" ").map(
+  //   (date: string) => date.charAt(0).toUpperCase() + date.slice(1)).join(" ");
+
+  const relativeDate = (inputDateTime.toRelativeCalendar() || "").split(" ").map(
```

Comment:
I think this is desired behaviour, but I originally thought we were talking about within 24 hours of now, although that implementation would likely be slightly confusing for times that were on the day before.